packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -180,7 +180,9 @@ export default function EntitiesSavedStates( { isOpen, closePanel } ) {
 
 	// Explicitly define this with no argument passed.  Using `closePanel` on
 	// its own will use the event object in place of the expected saved entities.
-	const dismissPanel = useCallback( () => closePanel(), [] );
+	// Do NOT useCallback, it will skip the 'this' bound callback used in the
+	// closePanel function.
+	const dismissPanel = () => closePanel();
```

Comment:
I don't think that's the issue here.

You are closing over `closePanel` which changes when the callback changes, so you need to add it as a dep.

Reply:
Ohhh yes!  100%.  So it was firing `closePanel` as it was at the state on mount, before the callback was set.  That makes sense.  Thank you!

Reply:
I am much happier now that the issue was the dep.  I was a bit confused late last night 😆 .

This should be fixed now as well.